src/resources.js

```diff
@@ -3,7 +3,7 @@ import transactions from './endpoints/transactions.js'
 import customers from './endpoints/customers.js'
 import transactionSplits from './endpoints/transaction_splits.js'
 import products from './endpoints/products.js'
-import subscriptions from './endpoints/subscriptions.js'
+import subscriptions from './endpoints/Subscriptions.js'
```

Comment:
Let's change the file to be `subscriptions` instead of `Subscriptions`

Reply:
We make it uniform.

Reply:
Cool. that will work too

Reply:
renamed file